android/build.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
 ext {
     travisBuild = System.getenv("TRAVIS") == "true"
     // allows for -Dpre-dex=false to be set
-    preDexEnabled = "true".equals(System.getProperty("pre-dex", "true"))
+    preDexEnabled = System.getProperty("pre-dex", "true") == "true"
```

Comment:
(To reviewers: This was an irrelevant change that I made. I made it for consistency's sake with the other variable in this block.)